src/bundles/CaseStudy/redux/modules/reducer.js

```diff
@@ -1,14 +1,15 @@
 import { combineReducers } from 'redux'
-import { combineForms } from 'react-redux-form';
+import { createForms } from 'react-redux-form';
 
 import casestudy from './casestudy';
 import errorMessage from './errorMessage';
+import form_options from './form_options';
 
 export default combineReducers({
   casestudy,
-  form_options: (state = { mode: 'add' }) => state,
-  form: combineForms({
-    caseStudy: {}
-  }, 'form'),
+  form_options,
   errorMessage,
+  ...createForms({
```

Comment:
Spread the forms at the root level.

Instead of having a state shape of:

```
{
  form: {
    caseStudy: { ... }
  }
}
```
it can now be

```
{
  caseStudyForm: { ... }
}
```